docker-compose.yml

```diff
@@ -8,6 +8,26 @@ services:
       POSTGRES_PASSWORD: secret
       POSTGRES_DB: prisma
     ports:
-      - "4224:5432"
+      - 4224:5432
     volumes:
       - ./db:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5      
+  fire-flags:
+    build: .
+    environment:
+      DATABASE_URL: postgresql://postgres:secret@pg:5432/prisma
+      # NEXT_AUTH_SECRET: RANDOM_STRING
+      # NEXTAUTH_URL: http://localhost:3000/api/auth
+    # command: >
+    #   sh -c "yarn prisma db push &&
+    #          yarn prisma generate &&
+    #          yarn run dev"
```

Comment:
why do we need these commented out lines please?

Reply:
safe to remove